### PR TITLE
feat(gh): init project command

### PR DESCRIPTION
**Pull request by esau-morais**
closes #963 

- [x] [gh project close](https://cli.github.com/manual/gh_project_close)
- [x] [gh project copy](https://cli.github.com/manual/gh_project_copy)
- [x] [gh project create](https://cli.github.com/manual/gh_project_create)
- [x] [gh project delete](https://cli.github.com/manual/gh_project_delete)
- [x] [gh project edit](https://cli.github.com/manual/gh_project_edit)
- [x] [gh project field-create](https://cli.github.com/manual/gh_project_field-create)
- [x] [gh project field-delete](https://cli.github.com/manual/gh_project_field-delete)
- [x] [gh project field-list](https://cli.github.com/manual/gh_project_field-list)
- [x] [gh project item-add](https://cli.github.com/manual/gh_project_item-add)
- [x] [gh project item-archive](https://cli.github.com/manual/gh_project_item-archive)
- [x] [gh project item-create](https://cli.github.com/manual/gh_project_item-create)
- [x] [gh project item-delete](https://cli.github.com/manual/gh_project_item-delete)
- [x] [gh project item-edit](https://cli.github.com/manual/gh_project_item-edit)
- [x] [gh project item-list](https://cli.github.com/manual/gh_project_item-list)
- [x] [gh project list](https://cli.github.com/manual/gh_project_list)
- [x] [gh project view](https://cli.github.com/manual/gh_project_view)

### CODE REVIEW

**Comment by esau-morais** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by esau-morais** [comment]
recheck

File: src/gh.ts [hunk 1]
```diff
@@ -2767,6 +2767,431 @@ For more information about output formatting flags, see 'gh help formatting'`,
...
+              args: { name: "owner" },
+            },
+            {
+              name: ["--readme"],
+              description: "New readme for the project",
+              args: { name: "readme" },
+            },
+            {
+              name: ["--visibility"],
+              description: "Change project visibility: {PUBLIC|PRIVATE}",
```
Comment (esau-morais) [inline src/gh.ts:2823]: remove unnecessary options in description

File: src/gh.ts [hunk 2]
```diff
@@ -2767,6 +2767,431 @@ For more information about output formatting flags, see 'gh help formatting'`,
...
+              args: { name: "visibility", suggestions: ["PUBLIC", "PRIVATE"] },
```
Comment (esau-morais) [inline src/gh.ts:2824]: make it case-insensitive or keep it as gh does?

File: src/gh.ts [hunk 3]
```diff
@@ -2767,6 +2767,431 @@ For more information about output formatting flags, see 'gh help formatting'`,
...
+          description: "Delete a project",
+          options: [
+            {
+              name: ["--owner"],
+              description: "Login of the owner. Use @me for the current user.",
+              args: { name: "owner" },
+            },
+            {
+              name: ["--format"],
+              description: "Output format: {json}",
```
Comment (esau-morais) [inline src/gh.ts:2869]: improve description or keep it as in the docs?

File: src/gh.ts [hunk 4]
```diff
@@ -2767,6 +2767,431 @@ For more information about output formatting flags, see 'gh help formatting'`,
...
+          options: [
+            {
+              name: "--name",
+              description: "Name of the field",
+              args: { name: "name" },
+            },
+            {
+              name: "--data-type",
+              description:
+                "DataType of the new field.: {TEXT|SINGLE_SELECT|DATE|NUMBER}",
```
Comment (esau-morais) [inline src/gh.ts:2958]: remove unnecessary options in description

File: src/gh.ts [hunk 5]
```diff
@@ -2767,6 +2767,431 @@ For more information about output formatting flags, see 'gh help formatting'`,
...
+            {
+              name: ["--owner"],
+              description: "Login of the owner. Use @me for the current user.",
+              args: { name: "owner" },
+            },
+            {
+              name: ["--format"],
+              description: "Output format: {json}",
+              args: { name: "format" },
+            },
```
Comment (esau-morais) [inline src/gh.ts:2887]: extract common options to a constant?

File: src/gh.ts (hunk 3 above)
Comment (grant0417) [reply src/gh.ts:2867]: Generally keeping them the same or close to docs is totally acceptable

File: src/gh.ts (hunk 5 above)
Comment (grant0417) [reply src/gh.ts]: You can, if its used >3 times I generally would.

**Comment by grant0417** [comment]
LGMT!

**Pull request merged**